app/helpers/application_helper.rb

```diff
@@ -537,14 +537,12 @@ def about_links
         :name => "about"
       }
     ]
-    if @community_customization && !@community_customization.how_to_use_page_content.blank?
```

Comment:
Oh, apparently this has been solved already on Flowdock so you can ignore this comment.